frontend/js/controllers/view_overview.js

```diff
@@ -38,7 +38,11 @@ angular
         // open file externally
         $scope.openDocument = function(article) {
             console.log("Open Document: " + article.filepath.value.value);
-            window.open(settings.files_url + article.filepath.value.value);
+            let nameEncode = encodeURIComponent("test_user");
```

Comment:
Anstatt der test-credentials kannst du die echten verwenden jetzt nach Ninos Änderungen: guck mal in der webservice.js